third_party/proto/ethermint/evm/v1/evm.proto

```diff
@@ -3,7 +3,7 @@ package ethermint.evm.v1;
 
 import "gogoproto/gogo.proto";
 
-option go_package = "github.com/evmos/ethermint/x/evm/types";
+option go_package = "github.com/evmos/evmos/v18/x/evm/types";
```

Comment:
todo - review all the proto files to make sure we have the same one as the upstream evmos v18 repo